LDM-602.tex

```diff
@@ -0,0 +1,646 @@
+\documentclass[SE,toc,lsstdraft]{lsstdoc}
+
+% We use commands to make it easy to find where parameter names and units
+% are defined in the tables, and to allow hyphenation.
+\newcommand{\paramname}[1]{\hspace{0pt}#1}
+\newcommand{\unitname}[1]{\hspace{0pt}#1}
+
+\setcounter{secnumdepth}{5}
+
+%% Retrieve date and model version
+\setDocUpstreamLocation{MagicDraw SysML}
+\setDocUpstreamVersion{72}
```

Comment:
I'm confused with what the above actually means. In two ways, actually.

First, are the requirements you're adding derived from MagicDraw or from LSE-61? The above implies the former, but below you are explicit that it is a flowdown from LSE-61. Of course, I understand that MagicDraw and LSE-61 ought to say basically the same thing, but I don't think that's good enough: I'd like there to be one canonical source of truth, from which everything is clearly derived.

Secondly, what is the meaning of the `DocUpstream` notation? Here, you seem to be using it to mean "this document is derived from some external source of truth", whereas [elsewhere](https://github.com/lsst/LDM-563/blob/ce3508dccd52e4a76edd0f1fac3960f1b44476dc/LDM-563.tex#L12) you're using it to mean "here is the repository which holds the source for this document". This doesn't seem to be discussed in the lsst-texmf documentation, but the [implementation](https://github.com/lsst/lsst-texmf/blob/master/texmf/tex/latex/lsst/lsstdoc.cls#L557) implies that it should be the latter (the "source location").

Reply:
This is a docgen of version 72 of the LDM-602 model. The Github repository is of secondary importance. The tex file is generated from MagicDraw and is never hand edited. I commit it to git for convenience (since it's easier to see what changed, and automated deployment can happen).

Reply:
by "source location" I meant "source of truth".

Reply:
PS All our requirements documents do this (including LSE-61).